dev-support/make_rc.sh

```diff
@@ -135,40 +135,40 @@ assembly() {
   svn add "${RATISVERSION}" || svn add "${RATISVERSION}/${RC#-}"
 }
 
-publish-git(){
+5-publish-git(){
   cd "$projectdir"
   git push apache "ratis-${RATISVERSION}${RC}"
 }
 
-publish-svn() {
+6-publish-svn() {
    cd "${SVNDISTDIR}"
   svn commit -m "Publish proposed version of the next Ratis release ${RATISVERSION}${RC}"
 }
 
-publish-mvn(){
+3-publish-mvn(){
```

Comment:
How about move this function to line 124?

Reply:
Left it here to clarify the actual diff.

Sure, we can move it.